LL/eoganachta_casil.ttl

```diff
@@ -1821,6 +1822,7 @@
     owl:sameAs <#Sechnasaig>.
 
 <#Rectabrat-22262fbc>
+    a irishTitles:Rí ;
     owl:sameAs <#Rechtabrat>;
     foaf:title "rig"@sga, "king"@en;
     foaf:nick "ind rige"@sga, "the king"@en;
```

Comment:
Really? To rephrase, _ind ríge_ does not mean "the king" and _in gaiscid_ does not mean "the spear". If we include these translations, users may think that our understanding of medieval Gaelic is rather superficial.